content/en/docs/tasks/observability/logs/access-log/index.md

```diff
@@ -61,7 +61,8 @@ All three of these parameters may also be configured via [install options](https
 1.  Send a request from `sleep` to `httpbin`:
 
     {{< text bash >}}
-    $ kubectl exec -it $(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}') -c sleep -- curl -v httpbin:8000/status/418
+    $ SLEEP_POD=$(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}')
+    $ kubectl exec -it "$SLEEP_POD" -c sleep -- curl -v httpbin:8000/status/418
```

Comment:
The `Before you begin` instructions already set an env variable `SOURCE_POD` for this, so you should really use that here.
```suggestion
    $ kubectl exec -it "$SOURCE_POD" -c sleep -- curl -v httpbin:8000/status/418
```